third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -2793,6 +2793,9 @@ def interpolate(self, method: str = "linear"):
                 Interpolation technique to use. Only 'linear' supported.
                 'linear': Ignore the index and treat the values as equally spaced.
                 This is the only method supported on MultiIndexes.
+                'index', 'values': use the actual numerical values of the index.
```

Comment:
Update the code samples too?

Reply:
added one more code sample. won't have space for every method as there will be 18 eventually